proto/proto/confirmed_block.proto

```diff
@@ -72,7 +72,18 @@ message TransactionError {
 
 message InnerInstructions {
     uint32 index = 1;
-    repeated CompiledInstruction instructions = 2;
+    repeated InnerInstruction instructions = 2;
+}
+
+message InnerInstruction {
```

Comment:
would be nice to have a blurb about how this would be used

Reply:
this folder is 99% copied from v1.17 solana labs repository